src/uct/ib/rc/base/rc_iface.c

```diff
@@ -1042,7 +1045,7 @@ void uct_rc_iface_vfs_refresh(uct_iface_h iface)
     uct_rc_ep_t *ep;
 
     /* Add iface resources */
-    uct_rc_iface_vfs_populate(rc_iface);
+    uct_rc_iface_vfs_populate(rc_iface, 1);
```

Comment:
can we call ucs_vfs_obj_add_ro_file(..., "cq_available") here instead of passing a flag to uct_rc_iface_vfs_populate?

Reply:
ok